README.md

```diff
@@ -78,64 +81,169 @@ You can run the examples by cloning the repo or copy pasting the below sample in
 ## Example 1 - CBS Authentication
```

Comment:
What does CBS stand for?

Reply:
`Claims Based Service`. Thanks for pointing that 👍